modules/libevent/2.1.12-stable.bcr.0/presubmit.yml

```diff
@@ -0,0 +1,36 @@
+matrix:
+  bazel:
+  - 8.x
+  linux_platform:
+  - debian11
+  - ubuntu2404
+  macos_platform:
+  - macos
+  windows_platform:
+  - windows
+tasks:
+  linux_presubmit:
+    name: Linux Presubmit
+    platform: ${{ linux_platform }}
+    bazel: ${{ bazel }}
+    test_targets:
+    - '@libevent//...'
```

Comment:
You could add `--process_headers_in_dependencies` and `--features=parse_headers` for better CI coverage of headers (assuming they are actually standalone).

Reply:
I get errors when I do that, and I'm not sure it is our job either. If this were my project, sure, I'd add every warning that makes sense. Adding warnings that aren't supported by the upstream project means we would have to patch any triggered warning and/or report upstream.